test/uk/gov/hmrc/plasticpackagingtax/registration/forms/liability/LiabilityStartDateSpec.scala

```diff
@@ -55,7 +47,7 @@ class LiabilityStartDateSpec extends AnyWordSpec with Matchers {
 
       "is exactly one year after tax liability starts (01-04-2022)" in {
 
-        val date = dateLowerLimit.plusYears(1)
+        val date = dateLowerLimit.minusYears(1)
```

Comment:
I think this should be `plusYears(1)` in order to test what is suggested by the test name.

Reply:
Changed this as well